README.md

```diff
@@ -1,3 +1,101 @@
-# STARC
+![Starc Logo](./img/starc.png)
```

Comment:
Can we make the logo a bit smaller? :) Or may be place it as a floating image with an introduction like [here](https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d)